snippets/elixir.snippets

```diff
@@ -183,6 +183,8 @@ snippet qu
 snippet beh
 	@behaviour ${1:Mix.Task}
 	${0}
+snippet p
+	|>
```

Comment:
How about `|> ${0}` so there's a space after as well? :)

Reply:
You just made that snippet 2x better! Thanks!